data-es2016plus.js

```diff
@@ -1156,8 +1187,8 @@ exports.tests = [
           firefox57: { val: "flagged", note_id: 'fx-shared-memory-spectre' },
           firefox72: firefox.nightly,
           firefox74: firefox.beta,
-          firefox77: {val: false, note_id: "fx-shared-memory-esr-disable" },
-          firefox79: {val: true, note_id: "fx-shared-memory-cors-isolation" },
+          firefox77: { val: false, note_id: "fx-shared-memory-esr-disable" },
+          firefox79: { val: true, note_id: "fx-shared-memory-cors-isolation" },
```

Comment:
also these

Reply:
Ah those are just my editor. I hate when it automatically formats things, so I don't know how it happened!